.circleci/config.yml

```diff
@@ -30,50 +30,253 @@ parameters:
+          at: .
+      - setup_remote_docker
+      - run:
+          name: Compute Tag for Docker images
+          command: |
+            # Compute tag with branch name, commmit SHA and timestamp. Then slugify it to be sure it can be handled by Docker
+            export TAG=$(echo "${CIRCLE_BRANCH:0:15}-${CIRCLE_SHA1:0:7}" | sed -E 's/[~^]+//g' | sed -E 's/[^a-zA-Z0-9]+/-/g' | sed -E 's/^-+|-+$//g' | tr A-Z a-z)
+            # Workaround for sharing this variable to the next steps
+            echo "export TAG=$TAG" >> $BASH_ENV
+            echo "Docker images will be tagged with: ${TAG}"
+            echo $TAG > docker-tag.txt
+      - run:
+          name: Build & Publish Portal UI Docker Image to Azure Registry
+          command: |
+            cp -fr docker/config .
+            cp -fr docker/run.sh .
+
+            export IMAGE_TAG=graviteeio.azurecr.io/apim-portal-ui:${TAG}
+            docker build -f docker/Dockerfile-dev -t ${IMAGE_TAG} .
+            echo $AZURE_DOCKER_REGISTRY_PASSWORD | docker login --username $AZURE_DOCKER_REGISTRY_USERNAME --password-stdin graviteeio.azurecr.io
+            docker push ${IMAGE_TAG}
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./docker-tag.txt
+
+  deploy-on-element-zero-cluster:
+    docker:
+      - image: mcr.microsoft.com/azure-cli:latest
+    resource_class: small
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Read tag of Docker images to deploy
+          command: |
+            export TAG=$(cat ./docker-tag.txt)
+            # Workaround for sharing this variable to the next steps
+            echo "export TAG=$TAG" >> $BASH_ENV
+            echo "Will deploy Docker images with tag: ${TAG}"
+      - run:
+          name: Install Helm
+          command: |
+            curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+            helm version
+      - run:
+          name: Install Kubectl
+          command: |
+            curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+            chmod +x ./kubectl
+            mv ./kubectl /usr/local/bin/kubectl
+            kubectl version --client=true
+      - run:
+          name: Helm upgrade and 🚀
+          command: |
+            az login --service-principal -u $AZURE_APPLICATION_ID --tenant $AZURE_TENANT -p $AZURE_APPLICATION_SECRET
+            az aks get-credentials --resource-group element-zero --name element-zero
+
+            helm upgrade --repo https://helm.gravitee.io \
+                        --install apim \
+                        -n apim \
+                        --reuse-values \
+                        apim3 \
+                        --set "portal.image.repository=graviteeio.azurecr.io/apim-portal-ui" \
+                        --set "portal.image.tag=$TAG"
+
+  purge-master-azure-registry:
```

Comment:
this job is defined but never used in a workflow. I think you forgot to add the `weekly-purge-azure-registry` workflow as in management-webui project

Reply:
Added